javascript/book.js

```diff
@@ -7,18 +7,39 @@ function updateLocalStorage(data) {
   localStorage.setItem('data', JSON.stringify(data));
 }
 
-let booksArray = JSON.parse(localStorage.getItem('data')) || [];
+/* eslint max-classes-per-file: ["error", 2] */
+class BooksArray extends Array {
+  static get() {
+    return Array;
+  }
+
+  removeBook(id) {
+    /* eslint-disable */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, Linter is a tool for identifying and reporting errors when you use standard rules. Kindly fix this instead of disabling it  👍🏻